robot/impl/resource_manager.go

```diff
@@ -845,10 +868,10 @@ func (manager *resourceManager) processResource(
 	ctx context.Context,
 	conf resource.Config,
 	gNode *resource.GraphNode,
-	r *localRobot,
+	lr *localRobot,
```

Comment:
sorry, meant to include this in [the separate prefactor](https://github.com/viamrobotics/rdk/pull/3961) - this variable name change is the only change in this function